runtime/common/ArgumentConversion.cpp

```diff
@@ -224,6 +225,11 @@ Value genConstant(OpBuilder &builder, cudaq::cc::StdvecType vecTy, void *p,
   auto eleTy = vecTy.getElementType();
   auto elePtrTy = cudaq::cc::PointerType::get(eleTy);
   auto eleSize = cudaq::opt::getDataSize(layout, eleTy);
+  if (isa<cudaq::cc::CharspanType>(eleTy)) {
+    // char span type (i.e. pauli word) is a `vector<char>`
```

Comment:
It shouldn't be, but that hasn't been fixed yet.

Reply:
ack.

Reply:
created an issue: https://github.com/NVIDIA/cuda-quantum/issues/2213